.github/workflows/build-onert-micro-arm.yml

```diff
@@ -43,4 +43,4 @@ jobs:
           mkdir build
           cd build
           cmake ../infra/onert-micro/
-          make -j$(nproc) luci_interpreter_micro_arm
+          for i in $(seq 1 5); do [ $i -gt 1 ] && sleep 5; make -j$(nproc) luci_interpreter_micro_arm && s=0 && break || s=$?; done; (exit $s)
```

Comment:
```suggestion
          for i in $(seq 1 5); do \
            [ $i -gt 1 ] && sleep 5; \
            make -j$(nproc) luci_interpreter_micro_arm && s=0 && break || s=$?; \
          done; \
          (exit $s)
```

Reply:
Q) if you are struggling with low memory why do you use `make -j$(nproc)` not `make -j1` ?